examples/index.html

```diff
@@ -17,6 +17,9 @@
   <script src="../node_modules/webgl-distort/dist/webgl-distort.js"></script>
   <script src="../node_modules/glfx/glfx.js"></script>
 
+  <!-- for EXIF geocode -->
+  <script type="text/javascript" src="../node_modules/exif-js/exif.js"></script>
```

Comment:
Do you prefer this to compiling it in? Thanks!

Reply:
@jywarren Should I add a browserify/requireJS support to this lib for the same?